.github/workflows/one-click-ga-beta-cloud-release.yml

```
@@ -0,0 +1,159 @@
+        uses: rgarcia-phi/json-to-variables@9835d537368468c4e4de5254dc3efeadda183793
+        with:
+          filename: "./.github/workflows/config.json"
+          prefix: config
+      - name: Test permissions
+        uses: ./.github/workflows/permissions
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY_OWNER: ${{ env.config_deployment_repository_owner }}
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Env Vars Configure
+        id: env_vars_configure
+        uses: ./.github/workflows/env-vars-configure
+        with:
+          CHANNEL: ${{ env.CHANNEL }}
+          REF: ${{ github.ref }}
+          POSTFIX: ''
+      - name: Bump Channel Version
+        id: bump_channel_version
+        uses: ./.github/workflows/bump-channel-version
+        with:
+          CLEAN_PACKAGE_VERSION: ${{ env.CLEAN_PACKAGE_VERSION }}
+          CHANNEL: ${{ env.CHANNEL }}
+          POSTFIX: ''
+          OVERRIDE_PACKAGE_VERSION: true
+      - name: Update Readme.txt
+        if: env.CHANNEL != 'cloud'
+        uses: ./.github/workflows/update-readme-txt
+        with:
+          README_TXT_PATH: $GITHUB_WORKSPACE/readme.txt
+          CHANNEL: ${{ env.CHANNEL }}
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+      - name: Get Release Name
+        if: env.CHANNEL != 'cloud'
+        uses : ./.github/workflows/get-release-name
+        with:
+          BUILD_ZIP_FILE_PATH: ${{ github.event.repository.name }}-${{ env.PACKAGE_VERSION }}.zip
+          PLUGIN_NAME: ${{ github.event.repository.name }}
+      - name: Generate changelog
+        uses: ./.github/workflows/generate-changelog
+        with:
+          TOKEN:  ${{ secrets.MAINTAIN_TOKEN }}
+          REPOSITORY: ${{ github.repository }}
+          HEAD_BRANCH_NAME: ${{ github.ref }}
+          BASE_TAG_NAME: ${{ env.PREVIOUS_TAG_SHA }}
+          GENERATE_EMPTY_CHANGELOG: true
+      - name: Create GitHub release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ env.PACKAGE_VERSION }}
+          target_commitish: ${{ env.CLEAN_REF }}
+          files: |
+            elementor-*.zip
+            ${{ env.CHANGELOG_FILE }}
+          prerelease: ${{ github.event.inputs.pre_release }}
+          body_path: ${{ env.CHANGELOG_FILE }}
+      - name: Publish to WordPress.org SVN
+        if: env.CHANNEL != 'cloud' && github.repository_owner == 'elementor' && github.event.inputs.pre_release == 'false' # We don't publish cloud to WordPress.org, ga, beta, dev are published.
```

If we don't have pre_releases this comparison is redundant.
```suggestion
        if: env.CHANNEL != 'cloud' && github.repository_owner == 'elementor'  # We don't publish cloud to WordPress.org, ga, beta, dev are published.
```

---

I've re-added prer-release. Lets remove it in next PR when everything will be tested